src/components/filterModal/statsList/BaseEditorLogarithmic.vue

```diff
@@ -51,29 +54,47 @@ export default {
         return {
             min: MIN,
             max: MAX,
-            selectedMin: this.preselectedMin || MIN,
-            selectedMax: this.preselectedMax || MAX,
+            currentMin: this.preselectedMin,
+            currentMax: this.preselectedMax,
+            selectedMin: this.closestLeftMark(this.preselectedMin),
+            selectedMax: this.closestRightMark(this.preselectedMax),
+            key: 0,
         };
     },
     methods: {
         addData() {
             this.onSubmit(this.selectedMin, this.selectedMax);
         },
         changeValues([min, max]) {
-            this.selectedMin = min;
-            this.selectedMax = max;
+            this.currentMin = min;
+            this.currentMax = max;
+            this.selectedMin = (min < this.preselectedMin ? this.closestLeftMark(this.preselectedMin) : this.closestLeftMark(min));
+            this.selectedMax = (max > this.preselectedMax ? this.closestRightMark(this.preselectedMax) : this.closestRightMark(max));
+        },
+        closestLeftMark(value) {
+            const arr = LOG_EDITOR_MARKS;
```

Comment:
This function works only once, if server sends any value not from LOG_EDITOR_MARKS. In all other cases you don't allow to save custom values, only from LOG_EDITOR_MARKS. It doesn't meet the requirements.
Don't call variables as arr, str as so on.